### DISCUSSION TITLE
Discord unban request

### DISCUSSION THREAD

**Discussion by comdar16** [Pryaxis/TShock#2479] [Q&A] [answered]
I want to get unbanned from TShock discord server. I apologize for all bad things i made.

My username and tag: comdar#1950

**Comment by hakusaro** [comment]
You aren't on the mod notes list (https://github.com/pryaxis/discord-mod-notes), so please send an appeal via email to argo@hey.com with exactly what you did, why you were banned, and uhh, an appeal.

**Reply by comdar16** [reply]
> You aren't on the mod notes list (https://github.com/pryaxis/discord-mod-notes), so please send an appeal via email to [argo@hey.com](mailto:argo@hey.com) with exactly what you did, why you were banned, and uhh, an appeal.

I m the guy who created anti-cheat topic.

**Reply by comdar16** [reply]
https://github.com/Pryaxis/TShock/discussions/2439

**Reply by hakusaro** [reply]
I have pinged relevant people about potentially unbanning you

**Reply by comdar16** [reply]
> I have pinged relevant people about potentially unbanning you

ok thanks

**Comment by hakusaro** [comment, marked as answer]
You're unbanned, as a last chance appeal.
